scripts/reportportal/rp_utils/rp_client.py

```
@@ -0,0 +1,270 @@
+        Returns:
+            List of launch dicts matching the filter.
+        """
+        url = self._api_url(path="launch")
+        params: dict[str, Any] = {
+            "filter.has.attributeKey": "BUNDLE",
+            "filter.cnt.attributeValue": bundle_prefix,
+        }
+        if since_days > 0:
+            since_ts = int((datetime.now(tz=UTC) - timedelta(days=since_days)).timestamp() * 1000)
+            params["filter.gte.startTime"] = since_ts
+            LOGGER.info(f"Filtering launches to last {since_days} days")
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)